src/Resources/MSGraph.Autorest/custom/New-AzADApplicationFederatedIdentityCredential.ps1

```diff
@@ -0,0 +1,141 @@
+
```

Comment:
Why is this custom required?

Reply:
These parameters are mandatory but generated optional, I didn't find any way to convert them by directive